From 6750cda26821f703b120ba5c925cc696200570d3 Mon Sep 17 00:00:00 2001
From: Maximilian Bosch <maximilian@mbosch.me>
Date: Sat, 26 Oct 2019 10:31:02 +0200
Subject: [PATCH] Remove coding annotations

Those used to be needed for Python <=3.5. With `pluggy` 0.13.0 this breaks
tests on newer python3 versions.
---
 mautrix_appservice/appservice.py  | 1 -
 mautrix_appservice/errors.py      | 1 -
 mautrix_appservice/intent_api.py  | 1 -
 mautrix_appservice/state_store.py | 1 -
 4 files changed, 4 deletions(-)

diff --git a/mautrix_appservice/appservice.py b/mautrix_appservice/appservice.py
index 3a141b1..47e37fe 100644
--- a/mautrix_appservice/appservice.py
+++ b/mautrix_appservice/appservice.py
@@ -1,4 +1,3 @@
-# -*- coding: future_fstrings -*-
 # Partly based on github.com/Cadair/python-appservice-framework (MIT license)
 from contextlib import contextmanager
 from typing import Optional, Callable, Awaitable, Union
diff --git a/mautrix_appservice/errors.py b/mautrix_appservice/errors.py
index 90d040f..702f541 100644
--- a/mautrix_appservice/errors.py
+++ b/mautrix_appservice/errors.py
@@ -1,4 +1,3 @@
-# -*- coding: future_fstrings -*-
 from typing import Optional
 
 
diff --git a/mautrix_appservice/intent_api.py b/mautrix_appservice/intent_api.py
index 4021bf8..7236cbb 100644
--- a/mautrix_appservice/intent_api.py
+++ b/mautrix_appservice/intent_api.py
@@ -1,4 +1,3 @@
-# -*- coding: future_fstrings -*-
 from urllib.parse import quote as urllib_quote
 from time import time
 from json.decoder import JSONDecodeError
diff --git a/mautrix_appservice/state_store.py b/mautrix_appservice/state_store.py
index 47bb970..6ebec2a 100644
--- a/mautrix_appservice/state_store.py
+++ b/mautrix_appservice/state_store.py
@@ -1,4 +1,3 @@
-# -*- coding: future_fstrings -*-
 from typing import Optional
 from abc import ABC, abstractmethod
 import json
-- 
2.23.0

